core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -101,32 +119,6 @@ public void start() {
         }
     }
 
-
-    @Override
-    public void stop() {
```

Comment:
Just noticed a bug - removing stop will mean that compose service containers and ambassador containers don't get destroyed at the end of a test. Instead they'll only be destroyed at JVM shutdown, which could be a bad thing for resource usage or even test correctness. Will reinstate, calling ContainerReaper to kill as necessary.